alvr/client_core/src/graphics/mod.rs

```diff
@@ -0,0 +1,9 @@
+use alvr_common::{Fov, Pose};
+
```

Comment:
A little bit empty rn planning on doing anything

Reply:
Of course, i have staging changes where i gradually move code inside here